### DISCUSSION TITLE
Cannot open multiple DataViews on the same table

### DISCUSSION THREAD

**Discussion by fushihara** [dbeaver/dbeaver#36380] [Q&A] [answered]
**Is the feature request related to a problem? Please describe.
Cannot open multiple DataViews on the same table

**Describe the solution you want.
Allow multiple DataViews to be displayed in the same table.
Or, I would like the ability to copy the SQL statement with the current sort and filter applied in the DataView.

![image](https://github.com/user-attachments/assets/a57ce193-878c-4246-929e-d266d23670b5)

**Comment by E1izabeth** [comment, marked as answer]
Hi, @fushihara 
You can press here to see full SQL 
![image](https://github.com/user-attachments/assets/082b26b3-f752-4302-ae58-03646cbb99a8)

![image](https://github.com/user-attachments/assets/5334a165-25c3-4399-b623-fcaa20efca4d)

**Comment by fushihara** [comment]
@E1izabeth 
It was already in perfect condition.
Thank you!
